Kinder/Classes/User.cs

```diff
@@ -12,39 +15,57 @@ public class User
     public int PlaceGive { get; set; }
     public int PlaceTake { get; set; }
 
-    public User() {}
-    
-    public static Boolean checkLogin(string Username, string Password) ///TO DO WITH FILES
+    public User() { }
+
+    static List<User> users = FileManager.getUsers();
+    public static Boolean CheckLogin(string Username, string Password)
     {
-        
-        if (Username == "admin" && Password == "admin") 
+        users = FileManager.getUsers();
+        foreach (User user in users)
         {
-            return true;
-        } else
-        {
-            return false;
+            if (Username == user.Username && Password == user.Password)
+            {
+                return true;
+            }
         }
+        return false;
     }
 
-    public static Boolean checkEmailAndPhoneNumber(string Email, string PhoneNumber) ///TO DO WITH FILES
+    public static Boolean CheckIfUserAlreadyExists(string Email, string PhoneNumber)
     {
-        if (Email == "dude@gmail.com" && PhoneNumber == "+12345")
+        users = FileManager.getUsers();
+        foreach (User user in users)
         {
-            return true;
+            if (Email == user.Email || PhoneNumber == user.PhoneNumber)
+            {
+                return false;
+            }
         }
-        else
+        return true;
+
+    }
+
+    public static Boolean CheckIfUsernameIsTaken(string Username)
+    {
+        users = FileManager.getUsers();
+        foreach (User user in users)
```

Comment:
I think this could be simplified, if you put method inside foreach, like: foreach (User user in FileManager.getUsers() )